### PR TITLE
(#1284) Fixed TempFolder close method

### PR DESCRIPTION
**Pull request by fanifieiev**
This PR is for issue [1284](https://github.com/yegor256/cactoos/issues/1284)

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#1307](https://github.com/yegor256/cactoos/issues/1307) is assigned to @fabriciofx[/z](https://www.0crat.com/u/fabriciofx), here is [why](https://www.0crat.com/footprint/C63314D6Z/6c60234c-293f-418f-a10e-e2ac0298e693); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

File: src/test/java/org/cactoos/io/TempFolderTest.java [hunk 1]
```diff
@@ -61,4 +67,51 @@ public void deletesDirectory() throws Exception {
...
+                    );
+                    return true;
+                }
+            )
+        ).exec(
+            new IterableOf<>(
+                "a", "b", "c", "d", "e"
+            )
+        );
+        new Assertion<>(
```
Comment (paulodamaso) [inline src/test/java/org/cactoos/io/TempFolderTest.java]: @fanifieiev Let's split this into two different test methods, one for each assertion

**Comment by fanifieiev** [comment]
@paulodamaso Please have a look again

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by 0crat** [comment]
@fabriciofx[/z](https://www.0crat.com/u/fabriciofx) resigned from [#1307](https://github.com/yegor256/cactoos/issues/1307), since the job is not in scope anymore

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 11@cb7eb55ecef8, thread: PQ-C63314D6Z
com.zerocracy.claims.ClaimIn[214] java.lang.IllegalArgumentException: Parameter "age" not found in "Order was canceled" among: [reason, priority, job, cause, _parent_sig, flow, voluntarily, login]
```

[0.51.6](https://github.com/zerocracy/farm): CID: [b324184b-6e49-4ee2-8bbc-135fdc970b22](https://www.0crat.com/footprint/C63314D6Z/b324184b-6e49-4ee2-8bbc-135fdc970b22), Type: "Order was canceled"

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)

**Pull request merged**